include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -265,10 +265,15 @@ namespace parallel
         const unsigned int coarse_cell_index) const override;
 
       /**
-       * Go through all locally owned cells and store the relations between
-       * cells and their CellStatus in the private member local_cell_relations.
+       * Go through all active cells which are locally owned and record how they
+       * will change in the private member vector local_cell_relations.
        *
-       * The stored vector will be ordered by the occurrence of cells.
+       * As no mesh refinement is supported at the moment for this class, all
```

Comment:
```suggestion
       * As no adaptive mesh refinement is supported at the moment for this class, all
```

(local refinement is possible; it only has to happen already during `create_triangulation()`)